honeybadgermpc/hbavss.py

```diff
@@ -69,10 +74,14 @@ def multicast(msg):
             raise HoneyBadgerMPCError("PolyCommit verification failed.")
 
         oks_recvd = 0
-        while oks_recvd < 2*self.t + 1:
-            _, avss_msg = await recv()  # First value is the `sid`
-            if avss_msg == HbAVSSMessageType.OK:
+        # to handle the case that a (malicious) node sends
+        # an OKmessage multiple times
+        ok_set = set()
```

Comment:
Same here, you don't need the `oks_recvd` anymore. Thanks for fixing this! :D

Reply:
Resolved.